lib/cn/csched_sp3.c

```diff
@@ -1370,11 +1370,17 @@ sp3_comp_thread_name(
     case CN_RULE_GARBAGE:
         r = "gb";
         break;
-    case CN_RULE_LENGTHK:
-        r = "lk";
+    case CN_RULE_LENGTH_MIN:
+        r = "ls";
         break;
-    case CN_RULE_LENGTHV:
-        r = "lv";
+    case CN_RULE_LENGTH_MAX:
+        r = "ll";
+        break;
+    case CN_RULE_LENGTH_VWLEN:
+        r = "lw";
```

Comment:
lvw instead of lw?

Reply:
We want them all to be two characters because the thread name limit is 16 characters.